dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
We can add a post-processor in SciGraph with a config saying: Tag nodes with these prefixes as anonymous. I think that'll do the trick.